nginx_stage/lib/nginx_stage.rb

```diff
@@ -145,6 +145,8 @@ def self.clean_nginx_env(user:)
       "RAILS_LOG_TO_STDOUT" => "true",
       # name change here because only OOD_* from apache is allowed through sudo rules
       "ALLOWED_HOSTS" => ENV['OOD_ALLOWED_HOSTS'],
+      # set the duplicate to keep clean_nginx_env impepodent
+      "OOD_ALLOWED_HOSTS" => ENV['OOD_ALLOWED_HOSTS'],
```

Comment:
One issue we were seeing was this function `clean_nginx_env` being called 2 or more times. First time, `ALLOWED_HOSTS` and `OOD_ALLOWED_HOSTS` erased. So, the next time it ran it then erased `ALLOWED_HOSTS` because `OOD_ALLOWED_HOSTS` was unset.